package.json

```diff
@@ -111,7 +111,7 @@
     "prop-types": "Since we use former ddb-react components that depend on prop-types we keep this. Should be removed when usage of prop-types is deprecated."
   },
   "dependencies": {
-    "@danskernesdigitalebibliotek/dpl-design-system": "^0.0.0-3bd678a68e0854369f27016877773f6878a1cb61",
+    "@danskernesdigitalebibliotek/dpl-design-system": "0.0.0-372e102a9ded64f39067eb2a13f7485342418a23",
```

Comment:
Would it be possible to always use latest release? Or do the design-system not change that offen?

It was this we talked about moving into npmjs.org

Reply:
The thing is that with npm/yarn you **can** run eg.: `yarn add @danskernesdigitalebibliotek/dpl-design-system@latest` but you'll end up with `@danskernesdigitalebibliotek/dpl-design-system@0.0.0-xxxxxxx`anyways. Apparently that is how it works opposed to composer.json where you literally have `@latest` specified in composer.json.
The result of that is that we have to remember run the `yarn add` command on a regular basis in order to fetch the latest HEAD.

Reply:
We may need to add that information to a readme to help others to not asking the same question :-)

Reply:
Diddly done :)

Reply:
It can be found in this commit: [Mention dpl-design-system npm package in README.md](https://github.com/danskernesdigitalebibliotek/dpl-react/pull/16/commits/40d5e31047122e1bf037ef8ca5a0335336e8127e)